api-report/telemetry-utils.api.md

```diff
@@ -45,7 +45,7 @@ export const disconnectedEventName = "disconnected";
 
 // @public
 export class EventEmitterWithErrorHandling<TEvent extends IEvent = IEvent> extends TypedEventEmitter<TEvent> {
-    constructor(errorHandler?: (eventName: EventEmitterEventType, error: any) => void);
+    constructor(errorHandler: (eventName: EventEmitterEventType, error: any) => void);
```

Comment:
I think this change was missed in [breaking changes documentation](https://github.com/microsoft/FluidFramework/blob/main/BREAKING.md)